src/Explorer/Notebook/useNotebook.ts

```diff
@@ -173,8 +175,10 @@ export const useNotebook: UseStore<NotebookState> = create((set, get) => ({
     isGithubTree ? set({ gitHubNotebooksContentRoot: root }) : set({ myNotebooksContentRoot: root });
   },
   initializeNotebooksTree: async (notebookManager: NotebookManager): Promise<void> => {
+    const notebookFolderName = userContext.features.phoenix === true ? "My Notebooks Scratch" : "My Notebooks";
```

Comment:
>Scratch [](http://example.com/codeflow?start=85&length=7)

not blocking this checkin- as discussed, let us rename it and hide it by default.